tests/compat.sh

```diff
@@ -599,7 +588,6 @@ setup_arguments()
     M_CLIENT_ARGS="server_port=$PORT server_addr=127.0.0.1 force_version=$MODE"
     O_CLIENT_ARGS="-connect localhost:$PORT -$O_MODE"
     G_CLIENT_ARGS="-p $PORT --debug 3 $G_MODE"
-    G_CLIENT_PRIO="NONE:$G_PRIO_MODE:+COMP-NULL:+CURVE-ALL:+SIGN-ALL"
```

Comment:
May I ask why `G_CLIENT_PRIO` is constructed but not used in this script? Is it redundant or just missed from the `CLIENT_CMD` of GNUTLS. @gilles-peskine-arm

Reply:
It doesn't seem to be useful, so yes, [I think we can remove it](https://github.com/Mbed-TLS/mbedtls/pull/6614/commits/ff88ece6feb4507a322704cfa63a163b3f57e928).